spec/models/chargeback_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe Chargeback do
+  # rpt.rpt_group     = @edit[:new][:rpt_group]
+  # rpt.rpt_type      = @edit[:new][:rpt_type]
+  # rpt.priority      = @edit[:new][:priority]
+  # rpt.categories    = @edit[:new][:categories]
+  # rpt.col_options   = @edit[:new][:col_options]
+
+  # rpt.order = @edit[:new][:sortby1].nil? ? nil : @edit[:new][:order]
+
+  let(:rpt) { MiqReport.new(:db => chargeback_model.name, :headers => [], :col_formats => []) }
+  let(:chargeback_model) { ChargebackVm }
+
+  # called from ReportController::Reports::Editor
+  # must be called on a subclass of Chargeback
+  describe "#set_chargeback_report_options" do
```

Comment:
Super-minor but since this is a class method:

```suggestion
  describe ".set_chargeback_report_options" do
```